apps/stocks/src/environments/environment.ts

```diff
@@ -6,8 +6,8 @@ import { StocksAppConfig } from '@coding-challenge/stocks/data-access-app-config
 
 export const environment: StocksAppConfig = {
   production: false,
-  apiKey: '',
-  apiURL: 'https://sandbox.iexapis.com'
+  apiKey: 'pk_e2111ff878704ffaa13a2c19e9d2c0f3',
+  apiURL: 'https://cloud.iexapis.com/'
```

Comment:
sandbox was giving error "Production tokens cannot be used in the sandbox environment. Please use https://cloud.iexpais.com"